src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
-import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const onLikeButtonClick = () => {
+    const updateChatEntry = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+    };
+    props.onUpdateLiked(updateChatEntry);
+  };
+
+  const heart = props.liked ? '❤️' : '🤍';
+  const alignment = props.sender === 'Vladimir'? 'chat-entry local' : 'chat-entry remote';
```

Comment:
Nice work setting up a ternary to decide which class name a particular bubble should be assigned to. Notice how both options mention `chat-entry`? We can DRY up this line by removing `chat-entry` and utilize string interpolation in the element's className.